setup.py

```diff
@@ -98,7 +98,7 @@
         "geojson>=2.5.0",
         "jsonschema>=3.2.0",
         "opm>=2023.10; sys_platform=='linux'",
-        "pandas>=1.1.5,<2.0",
```

Comment:
I just checked and webviz-subsurface is not compatible with `pandas-3.0.0dev0` so to keep the wheels spinning I suggest we have `pandas>=1.1.5,<3`.

Reply:
:+1: